tests/src/test/java/alluxio/server/ft/journal/JournalShutdownIntegrationTest.java

```diff
@@ -137,9 +137,9 @@ public void multiMasterJournalStopIntegration() throws Exception {
     MultiProcessCluster cluster =
         MultiProcessCluster.newBuilder(PortCoordination.JOURNAL_STOP_MULTI_MASTER)
             .setClusterName("multiMasterJournalStopIntegration")
+            .setDeployMode(MultiProcessCluster.getDeployMode(TEST_NUM_MASTERS))
```

Comment:
could the cluster figure out the deploy mode on its own based on how many masters are configured and whether embedded journal is configured?

Reply:
Now if users do not give their desired deploy mode, we get deploy mode according to the num of masters and master journal type by default.